guidelines/_sources/04_report-prompts.tex

```diff
@@ -4,7 +4,10 @@
 
 \subsection{Report Prompts and their Development}
 
+\comment{I found it a bit hard to understand when to use \textbf{MUST} and \textbf{SHOULD} in this section. I think we need to clarify this. Sorry, computer networks lecturer's taking over, but we could follow the RFC2119 that defines requirements levels in Internet protocol specifications (https://datatracker.ietf.org/doc/html/rfc2119). Something along this line?  \textbf{MUST}: An absolute requirement for reproducibility.  \textbf{SHOULD}: A strongly recommended practice that may be omitted with justification. \textbf{MAY}: An optional practice.}
```

Comment:
@sbaltes ~Why is Fabio telling me what to do?~ Do you have any guidance on this? I assume this is not the only guideline where we could distinguish between absolute requirements, recommended practices, and optional things. Do we want such a distinction, or should I simplify this?